nimbus-consensus/src/lib.rs

```diff
@@ -320,28 +320,13 @@ where
 		relay_parent: PHash,
 		validation_data: &PersistedValidationData,
 	) -> Option<ParachainCandidate<B>> {
-		// Determine if runtime change
-		let runtime_upgraded = if *parent.number() > sp_runtime::traits::Zero::zero() {
```

Comment:
how about we split this change and the toolchain change into its own PR and try to merge the rest to usptream nimbus ?

Reply:
:100: agreed, it's already two different branches, i'll create a separate nimbus fork PR that has only the functional changes and points agains the 0.9.26 branch instead of main